### PR TITLE
Feature year picker question

### LINKED ISSUE #1012: Add year selector view to DynamicTabAdapter

### PR DESCRIPTION
**Pull request by manuelplazaspalacio**
Close #1012

### CODE REVIEW

File: app/src/main/java/org/eyeseetea/malariacare/views/YearPicker.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.NumberPicker;
+
+import org.eyeseetea.malariacare.R;
+import org.joda.time.DateTime;
+
+
+public class YearPicker extends DialogFragment {
```
Comment (xurxodev) [inline app/src/main/java/org/eyeseetea/malariacare/views/YearPicker.java]: @manuelplazaspalacio YearPicker should be a EyeSeeTea SDK control and with customizable font as other controls that exists in SDK

File: app/src/main/res/layout/fragment_year_picker.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (xurxodev) [inline app/src/main/res/layout/fragment_year_picker.xml]: @manuelplazaspalacio Can you refactor year dialog view style according to look and feel of the app?

**Comment by manuelplazaspalacio** [comment]
@xurxodev Request changes done. Not changing look and feel to much but @ifoche says that it's ok like that.

**Comment by manuelplazaspalacio** [comment]
@xurxodev Solved read mode bug

**Pull request merged**